### PR TITLE
adds SpecifiedLegalOrganization

### PR DESCRIPTION
**Pull request by MAKOMO**

### CODE REVIEW

File: drafthorse/models/party.py [hunk 1]
```diff
@@ -59,6 +59,22 @@ class Meta:
         tag = "EmailURIUniversalCommunication"
 
 
+class LegalOrganization(Element):
+    id = StringField(NS_RAM, "ID", required=False, profile=BASIC)
```
Comment (MAKOMO) [inline drafthorse/models/party.py:63]: Maybe this should be an IDField as it reads type="udt:IDType" in the spec which allows for an optional schemeID. There are other places where StringFields might need to be promoted to IDFields.

File: drafthorse/models/party.py [hunk 2]
```diff
@@ -59,6 +59,22 @@ class Meta:
...
+    trade_name = StringField(
+        NS_RAM,
+        "TradingBusinessName",
+        required=False,
+        profile=BASIC,
+        _d="Name, unter dem der Käufers bekannt ist",
```
Comment (bwurst) [inline drafthorse/models/party.py:69]: Can you please fix the typo "Käufers" here? ;-)

File: drafthorse/models/party.py [hunk 3]
```diff
@@ -75,31 +91,42 @@ class Meta:
...
-        NS_RAM, "ID", required=False, profile=COMFORT, _d="Identifier des Verkäufers"
+        NS_RAM, "ID", required=False, profile=COMFORT, _d="Kennung des Handelspartners"
     )
     global_id = MultiIDField(
         NS_RAM,
         "GlobalID",
         required=False,
         profile=COMFORT,
-        _d="Globaler Identifier des Verkäufers",
+        _d="Globaler Kennung des Handelspartners",
```
Comment (raphaelm) [inline drafthorse/models/party.py:101]:
```suggestion
        _d="Globale Kennung des Handelspartners",
```

File: drafthorse/models/party.py [hunk 4]
```diff
@@ -59,6 +59,22 @@ class Meta:
...
 
 
+class LegalOrganization(Element):
+    id = IDField(NS_RAM, "ID", required=False, profile=BASIC)
+    trade_name = StringField(
+        NS_RAM,
+        "TradingBusinessName",
+        required=False,
+        profile=BASIC,
+        _d="Firmenname, sofern abweichend vom Name",
```
Comment (raphaelm) [inline drafthorse/models/party.py:69]:
```suggestion
        _d="Firmenname, sofern abweichend vom Namen",
```

**Review by raphaelm — APPROVED** [review]

**Pull request merged**